data-transformers/transformers/statsTransformer.py

```diff
@@ -21,6 +21,7 @@
 trialID = sys.argv[5]
 experimentID = trialID.split("_")[0]
 containerPropertiesID = sys.argv[7]
+containerID = filePath.split("/")[-1].split("_")[0]
```

Comment:
Ok, lets proceed as you propose. It makes sense and the chance of a conflict seems to be practically near to zero, especially in out current environment. I updated the following issue so that we have a starting point to improve this functionality and reduce to zero the likelyhood of a conflict: https://github.com/benchflow/collectors/issues/46